### ISSUE TITLE
Internal compiler error on variable named "exports"

### ISSUE THREAD

**Issue by martine** [google/closure-compiler#1714]
This input:
```js
goog.module('a.b');

let exports = 1;
```

Produces this output:
```sh
$ java -jar /.../node_modules/google-closure-compiler/compiler.jar --js st/sickle.js --js_output_file=bundle.js --warning_level=VERBOSE --language_in=ECMASCRIPT6_STRICT --compilation_level=ADVANCED_OPTIMIZATIONS 
java.lang.RuntimeException: INTERNAL COMPILER ERROR.
Please report this problem.

Invalid left-hand side: GETPROP 3 [source_file: st/sickle.js]
  Node(BLOCK): st/sickle.js:1:0
goog.module('a.b');
  Parent(FUNCTION ): st/sickle.js:1:0
goog.module('a.b');

	at com.google.common.base.Preconditions.checkState(Preconditions.java:197)
	at com.google.javascript.jscomp.Es6SyntacticScopeCreator.declareLHS(Es6SyntacticScopeCreator.java:141)
	at com.google.javascript.jscomp.Es6SyntacticScopeCreator.scanVars(Es6SyntacticScopeCreator.java:163)
	at com.google.javascript.jscomp.Es6SyntacticScopeCreator.scanVars(Es6SyntacticScopeCreator.java:220)
	at com.google.javascript.jscomp.Es6SyntacticScopeCreator.scanRoot(Es6SyntacticScopeCreator.java:107)
	at com.google.javascript.jscomp.Es6SyntacticScopeCreator.createScope(Es6SyntacticScopeCreator.java:65)
	at com.google.javascript.jscomp.NodeTraversal.getScope(NodeTraversal.java:751)
	at com.google.javascript.jscomp.ReferenceCollectingCallback.enterScope(ReferenceCollectingCallback.java:235)
	at com.google.javascript.jscomp.NodeTraversal.pushScope(NodeTraversal.java:688)
	at com.google.javascript.jscomp.NodeTraversal.traverseBlockScope(NodeTraversal.java:662)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:609)
	at com.google.javascript.jscomp.NodeTraversal.traverseFunction(NodeTraversal.java:655)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:607)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:615)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:615)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:615)
	at com.google.javascript.jscomp.NodeTraversal.traverseBranch(NodeTraversal.java:615)
	at com.google.javascript.jscomp.NodeTraversal.traverseRoots(NodeTraversal.java:316)
	at com.google.javascript.jscomp.NodeTraversal.traverseRoots(NodeTraversal.java:576)
	at com.google.javascript.jscomp.ReferenceCollectingCallback.process(ReferenceCollectingCallback.java:115)
	at com.google.javascript.jscomp.VariableReferenceCheck.process(VariableReferenceCheck.java:90)
	at com.google.javascript.jscomp.PhaseOptimizer$NamedPass.process(PhaseOptimizer.java:285)
	at com.google.javascript.jscomp.PhaseOptimizer.process(PhaseOptimizer.java:217)
	at com.google.javascript.jscomp.Compiler.check(Compiler.java:777)
	at com.google.javascript.jscomp.Compiler.compileInternal(Compiler.java:693)
	at com.google.javascript.jscomp.Compiler.access$000(Compiler.java:89)
	at com.google.javascript.jscomp.Compiler$2.call(Compiler.java:651)
	at com.google.javascript.jscomp.Compiler$2.call(Compiler.java:648)
	at com.google.javascript.jscomp.CompilerExecutor$2.call(CompilerExecutor.java:93)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.IllegalStateException: Invalid left-hand side: GETPROP 3 [source_file: st/sickle.js]
	... 33 more
```

**Comment by martine** [comment]
```sh
$ java -jar /home/evanm/projects/sickle/node_modules/google-closure-compiler/compiler.jar --version
Closure Compiler (http://github.com/google/closure-compiler)
Version: v20151216
Built on: 2015/12/18 14:23
```

**Comment by Dominator008** [comment]
This has been fixed in the latest release. Please upgrade to v20160308.

**Comment by martine** [comment]
Thanks!

**Comment by Dominator008** [comment]
@martine My bad, this was fixed in cc05a011a002c52857335d9a72f826a96e3caf3a, which hasn't been released yet.
